src/bindings/CMakeLists.txt

```diff
@@ -34,3 +34,10 @@ if (FINDEX GREATER -1)
 		add_subdirectory (gi)
 	endif ()
 endif ()
+
+list (FIND BINDINGS "gsettings" FINDEX)
```

Comment:
True, the current setup was made on the idea to also make it possible to build if gelektra was already installed on the system, but by adding glib to the binding list this idea is deprecated anyway. I will change it.